terraform/modules/networking/vpc.tf

```diff
@@ -0,0 +1,20 @@
+module "vpc" {
+  providers = {
+    # https://github.com/terraform-aws-modules/terraform-aws-vpc/issues/625
+    aws = aws.no-default-tags
+  }
+
+  source = "github.com/hmrc/terraform-aws-vpc?ref=v3.19.0"
+
+  name = local.vpc_name
+
+  cidr = "10.0.0.0/16"
+
+  azs             = ["eu-west-2a", "eu-west-2b", "eu-west-2c"]
+  private_subnets = ["10.0.1.0/24", "10.0.2.0/24", "10.0.3.0/24"]
```

Comment:
Not a criticism of this, but be nice if we had appropriate tagging on subnets which we could just use a datasource to pull instead. This works as is, maybe something we can look at with tagging compliance work